bin/mr_bump

```diff
@@ -55,6 +55,12 @@ loop do
     MrBump.file_prepend('CHANGELOG.md', md_changes)
     system 'nano CHANGELOG.md'
     loop do
+      log = File.open('CHANGELOG.md', 'r').read
+      md_changes = log.split("\n\n").first.split("\n").drop(1)
```

Comment:
That sounds cool. If we wanted to add links to the changelog we would need to make sure we add the links after its been sent to slack, as the formatting standards for links are different. We could still do that with a temp file though.